api/scpca_portal/metadata_file.py

```diff
@@ -111,4 +118,6 @@ def write_metadata_dicts(list_of_dicts: List[Dict], output_file_path: str, **kwa
     with open(output_file_path, "w", newline="") as raw_file:
         csv_writer = csv.DictWriter(raw_file, **kwargs)
         csv_writer.writeheader()
-        csv_writer.writerows(sorted_list_of_dicts)
+        # Make sure to format metadata values when writing each row
```

Comment:
```suggestion
```

This comment isn't particularly helpful so I think we should remove it.

Reply:
See the commit at aff961cae29896428fdd0fa76b1ac61dfa35a2bf.